src/main/java/com/cloudogu/smeagol/wiki/infrastructure/StaticContentController.java

```diff
@@ -0,0 +1,57 @@
+package com.cloudogu.smeagol.wiki.infrastructure;
+
+import com.cloudogu.smeagol.wiki.domain.Path;
+import com.cloudogu.smeagol.wiki.domain.WikiId;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Serves static content, such as images, from a wiki repository.
+ *
+ */
+@RestController
+@RequestMapping(StaticContentController.MAPPING)
+public class StaticContentController {
+
+    static final String MAPPING = "/api/v1/repositories/{repositoryId}/branches/{branch}/static";
+
+    private final WildcardPathExtractor pathExtractor;
+    private final GitClientProvider gitClientProvider;
+
+    @Autowired
+    public StaticContentController(WildcardPathExtractor pathExtractor, GitClientProvider gitClientProvider) {
+        this.pathExtractor = pathExtractor;
+        this.gitClientProvider = gitClientProvider;
+    }
+
+    @RequestMapping("**")
+    public ResponseEntity<FileSystemResource> findStaticContent(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: org.eclipse.jgit.api.errors.GitAPIException, java.io.IOException [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1160)